gulpfile.js

```diff
@@ -41,7 +32,19 @@ gulp.task('install-azure-account', () => {
             }))
             .pipe(gulp.dest(extensionPath));
     } else {
-        console.log('Azure Account extension already installed.');
+        // We don't use console.log because we need to signal completion of this async task to gulp
+        return gulp.src('package.json')
```

Comment:
Haven't used gulp much, but the piping seems unnecessary.  Can't you do something like return Promise.resolve('Azure account already installed.')?

Reply:
I was trying to maintain the same return as the other case, for homogeneity. 

Gulp reports the status correctly, but the message seems missing when using `Promise.resolve("Azure account already installed.")`
![image](https://user-images.githubusercontent.com/30420973/49408678-1646a280-f712-11e8-84f7-b4c3646748cb.png)

Reply:
Suggest:
```
console.log('Azure Account extension already installed.');
return Promise.resolve();
```

Reply:
Yep. There's no reason to complicate (and do something weird, like pull in a file unrelated to the logic) in order to return the same type of value.